docs/before/accounts.md

```diff
@@ -0,0 +1,101 @@
+
+!!! note "Quicklinks"
+    - **Account** - [Create a UNIX account][6]
+    - **Computing** - [Access and Resources][1]
+    - **Data Analysis** - for [LCLS-I][2] and for [LCLS-II][3]
+    - **FFB docs** - [PCDS docs][8]
+    - **S3DF docs** - ["Running at S3DF" for LCLS][4] and [official S3DF documentation][5].
+
+
+# Step 1 - Get a computer account
+
+To access SLAC computing resources, you need a valid SLAC UNIX account. A detailed description of the steps required to create one can be found [here][6].
+
+!!! note "Enabling access to the S3DF"
```

Comment:
We should probably add this to the glossary. If I don't know what the S3DF is this might be confusing to me. Why should I care about it?

We can also link to the glossary

Reply:
Done